src/lib/yang/data.lua

```diff
@@ -243,10 +243,10 @@ local function bit_validator(range, f) return f end
 local function enum_validator(enums, f)
    if not enums or #enums == 0 then return f end
```

Comment:
i think this check will no longer work as enums is a string-keyed table instead of an array